panos/panorama.py

```diff
@@ -751,6 +789,34 @@ def get_vm_auth_keys(self):
 
         return ans
 
+    def get_device_group_hierarchy(self):
```

Comment:
The method's name implies getting a hierarchy, which to me implies a nested dict. But as you say in the docstring it's a flat dict where the value is the DG parent. Perhaps another name could better represent this expectation?